packages/teams-js/src/public/app.ts

```diff
@@ -46,10 +46,14 @@ const initializationTimeoutInMs = 5000;
 
 const appLogger = getLogger('app');
 
-export function appInitializeHelper(apiVersionTag: string, validMessageOrigins?: string[]): Promise<void> {
+export function appInitializeHelper(
+  apiVersionTag: string,
+  validMessageOrigins?: string[],
+  customMessageIdStarter?: number,
```

Comment:
Is this something we really want to enable the entire world to do?

Reply:
no just for 3pcxp in a beta release. (correct me if I am wrong @erinha)

Reply:
Correct. This would not be something that anyone outside of this very specific use case should do. I don't think we'd ever release this in a prod version, for example. So far, we are in a prototyping world. (In fact, we should probably make this a draft PR.)

Reply:
We should also look at ways to keep this parameter from showing up in docs (take a look at the resources parameter for `authentication.getAuthToken` as an example). @lakhveerk